bedrock/firefox/templates/firefox/win10_variants/variant-2-4.html

```diff
@@ -0,0 +1,23 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/win10-welcome.html" %}
+
+{% block page_title %}Make Firefox your default browser now{% endblock %}
```

Comment:
This headline will be shown to the user if Firefox is already their default browser, is that ok? If memory serves me correctly, I think in the last test we worked around this by making the headline conditional.